components/omega/configs/Default.yml

```diff
@@ -2,7 +2,7 @@ Omega:
    TimeManagement:
       StartTime: 0001-01-01_00:00:00
       StopTime: 0001-01-01_02:00:00
-      RunDuration: none #0000_02:00:00.000
+      RunDuration: none
```

Comment:
@brian-oneill, I know this comment seems helpful (indicating the default run duration based on the default start and stop time) but the way the yaml file gets parsed, the comments persist even when the config options get replaced.  So the only comments in the yaml file need to be general explanations of the config options, not anything related to specific, default values.

Reply:
This is an artifact of when I was doing some testing, didn't mean for it to end up in the PR, agree it should be removed.